### PR TITLE
Prometheus sizing workload

### PR DESCRIPTION
**Pull request by rsevilla87**
### Description

New prometheus sizing workload. Two variants:

- Static: "Fills" worker nodes with 250 pods and waits 125 minutes before collecting metrics
- Churning: Creates 50 pods per node by default, divided in 8 namespaces by default, and rotates (Deletes that namespace and creates a new one) one these namespace each 15 minutes (by default as well).

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

**Review by whitleykeith — APPROVED** [review]
LGTM

**Review by chaitanyaenr — COMMENTED** [review]
Minor nits

File: workloads/prometheus-sizing/env.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+export BURST=${BURSTS:-40}
+export CLEANUP_WHEN_FINISH=${CLEANUP_WHEN_FINISH:-true}
+export ENABLE_INDEXING=${ENABLE_INDEXING:-true}
+export ES_SERVER=${ES_SERVER:-https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443}
+export ES_INDEX=${ES_INDEX:-ripsaw-kube-burner}
+
+# prometheus-sizing-static specific
+export JOB_PAUSE=${JOB_PAUSE:-125m}
+
+# prometheus-sizing-churning specific
```
[the remark this answers is not in the document]
Comment (rsevilla87) [reply workloads/prometheus-sizing/env.sh:13]: Yeah, that was a mistake. Updating them

File: workloads/prometheus-sizing/metrics.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+metrics:
```
[the remark this answers is not in the document]
Comment (rsevilla87) [reply workloads/prometheus-sizing/metrics.yaml:1]: We don't query prometheus here, I think it's not worth to capture these metric as this is not the main goal from this workload.

**Review by jtaleric — APPROVED** [review]
LGTM!

**Pull request merged**